R/ci.R

```diff
@@ -2,7 +2,7 @@
 #'
 #' @description
 #'
-#' \lifecycle{soft-deprecated}
+#' `r lifecycle::badge("soft-deprecated")`
```

Comment:
It's not clear to me from #1303 that, if we used usethis this itself for lifecycle setup, we would have access to a "soft-deprecated" badge (although we seem to have the svg in `man/figures`).